packages/builder/src/steps/router.ts

```diff
@@ -111,16 +111,18 @@ const routerStep = {
         throw new Error(`contract not found: ${n}`);
       }
 
+      const contractName = n.replace('.', '_'); // Use step name, and replace '.' in case is pointing to an import.
```

Comment:
This fixes a bug where it was not possible to render a Router that included contracts from an imported package 🐛